qualtran/bloqs/block_encoding/block_encoding_base.py

```diff
@@ -93,6 +93,11 @@ def epsilon(self) -> SymbolicFloat:
         """The precision to which the block encoding is to be prepared."""
         raise NotImplementedError
 
+    @property
```

Comment:
See my comment in `QubitizationWalkOperator` --- given the removal of the other `_registers` methods from the `BlockEncoding` interface, could we add `control_registers` only to `LCUBlockEncoding` and `LCUBlockEncodingZeroState`?

We can then revisit how to add control registers to the block encoding quantum interface.

Reply:
Done.